rust_src/build.rs

```diff
@@ -349,10 +349,20 @@ fn env_var(name: &str) -> String {
 }
 
 // What to ignore when walking the list of files
+#[cfg(feature = "libvterm")]
 fn ignore(path: &str) -> bool {
     path == "" || path.starts_with('.') || path == "lib.rs" || path == "functions.rs"
 }
 
+#[cfg(not(feature = "libvterm"))]
```

Comment:
Instead of repeating the base `ignore` maybe you should make a `base_ignore` which is all that the vterm version uses and the non-vterm version is it plus vterm.rs.